stacks/environment-aws/main.tf

```diff
@@ -35,6 +35,8 @@ provider "helm" {
     "spec.template.spec.affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].values[0]=preemptible",
     "spec.template.spec.tolerations[0].key=${var.essential_taint_key}",
     "spec.template.spec.tolerations[0].operator=Exists",
+    "spec.template.spec.containers[0].resources.limits.memory=512Mi",
+    "spec.template.spec.containers[0].resources.limits.memory=512Mi",
```

Comment:
`256Mi` was working for me in sandbox, any reason this needs to be `512`?

Reply:
That's a mistake, my bad

Reply:
I think 256 is a good value for this though, that's what I had to bump it to in sandbox to avoid it getting OOMKilled

Reply:
It's currently 512Mi in sandbox. Would you prefer 256? I made the mistake of setting the limit twice and not the request

Reply:
Sorry, I meant 256 for requests, 512 for limits

Reply:
Gotcha, I'll push up those changes right now